frontend/src/i18n/locales/features/admin-form/settings/email-notifications/en-sg.ts

```diff
@@ -0,0 +1,39 @@
+import { OGP_PLUMBER } from '~constants/links'
+
+export const enSG = {
+  title: 'Email notifications',
+  header: {
+    closeFormFirst:
+      'To change email recipients, close your form to new responses.',
+    noEmailsForPaymentForms: `Email notifications for payment forms are not available in FormSG. You can configure them using [Plumber](${OGP_PLUMBER}).`,
```

Comment:
Would prefer for the url to be passed in from the component instead. 

Want to avoid having non-i18n imports in locale.

Reply:
Okay, let me rework this